st2common/st2common/util/param.py

```diff
@@ -216,7 +216,8 @@ def _cast_params_from(params, context, schemas):
     return result
 
 
-def render_live_params(runner_parameters, action_parameters, params, action_context):
+def render_live_params(runner_parameters, action_parameters, params, action_context,
+        additional_contexts={}):
```

Comment:
I missed that earlier - we need to default this to `None` and do `additional_contexts = additional_contexts or {}` since it's mutable value and mutable default values cause issues and usually unexpected behavior.